presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -90,6 +90,25 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
+-----------------------------------
+
+With the ``data_compression`` table property you can specify the data compression
+policy for the SQL Server tables. Date compression policy can either be
```

Comment:
Valid policies are ..

Reply:
Maybe also add a link to more info about that in the SQL Server docs somewhere..